src/primitives/block.h

```diff
@@ -66,7 +66,7 @@ class CBlockHeader : public CPureBlockHeader
 class CBlock : public CBlockHeader
 {
 public:
-    // network and disk
```

Comment:
I can see you did a "replace all" but maybe we should not replace the word "network" with "netwow"

Reply:
done